data/mods/MindOverMatter/monsters/feral_psychics.json

```
@@ -330,6 +330,224 @@
       "ALL_SEEING"
     ]
   },
+  {
+    "id": "mon_feral_human_photokin",
+    "type": "MONSTER",
+    "name": "feral radical",
```

```suggestion
    "name": "feral radiant",
```
Since the feral is using photokinesis, I think "radiant" fits a bit more, since it's associated with light.